examples/handlebars/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@frctl/handlebars-example",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "fractal start --sync",
+    "build": "fractal build",
+    "test": "jest",
+    "test:dev": "npm run test -- --watch"
+  },
+  "dependencies": {
+    "@frctl/fractal": "^1.3.0",
+    "@frctl/handlebars": "^1.2.0",
+    "@frctl/mandelbrot": "^1.4.0"
+  },
+  "devDependencies": {
+    "jest": "^26.0.1",
```

Comment:
I have a bit of an issue with this, but it's one that we may need to chat about in Slack.

We don't have many tests, but where we do - they're using Mocha/Chai/Sinon. The reason I'm not saying "Nope, we should change this" and "We should chat" instead, is that I'd REALLY like to migrate all our tests to Jest instead to reduce the amount of dependencies we need (as Jest literally covers everything we need).

Reply:
I'm not against switching to Jest completely, but I'd wager that could be another issue/pr for another day. We don't exactly lose anything by having multiple test runners for a while. So if we could write this (migrating away from mocha/chai/sinon) up as an issue, I wouldn't block this PR.

Reply:
Yeah I'm also in favor of going full Jest, but I'd like to go step by step on this one as Mihkel said.